crates/rpc/src/jsonrpc/websocket/data.rs

```diff
@@ -129,6 +142,7 @@ impl Serialize for ResponseEvent {
                 RpcResponse::invalid_params(id.into(), e.clone()).serialize(serializer)
             }
             ResponseEvent::Header(header) => header.serialize(serializer),
+            ResponseEvent::Event(event) => event.serialize(serializer),
```

Comment:
One potential issue here is that we're exposing the common `Event` type in the response here. Unfortunately that structure does not quite match [EMITTED_EVENT](https://github.com/starkware-libs/starknet-specs/blob/76bdde23c7dae370a3340e40f7ca2ef2520e75b9/api/starknet_api_openrpc.json#L928) in the JSON-RPC specification: it's missing the context (`transaction_hash`, `block_hash` and `block_number`). Should we aim for a response that matches the one you'd get via `starknet_getEvents`?